3_visualize/src/report-wqp-salinity-data.Rmd

```diff
@@ -13,7 +13,7 @@ output: html_document
 ##------------ WQP ------------##
```

Comment:
Should we re-render the resulting .html file? We currently comment out the p3_SC_report target because if issues getting it to load to S3. We could render it, commit the file to GitHub, and then comment out again.

Reply:
My vote would be to leave as is since the html doesn't render nicely on GH anyway. I only updated this file so that it would render if a user were working with it on their local clone. I'm fine with leaving that target commented out, though.